src/components/ListTemplate/ColumnTemplates/DefaultColumn/DefaultColumn.js

```diff
@@ -0,0 +1,53 @@
+import React, { memo } from 'react';
+import HighLight from 'react-highlighter';
+import PropTypes from 'prop-types';
+
+import { AppIcon } from '@folio/stripes/core';
+
+import sharedCss from '../../../../shared.css';
+
+export const DefaultColumn = memo(props => {
+  const {
+    value,
+    className,
+    iconKey,
+    searchTerm,
+  } = props;
+
+  const recordContent = () => {
+    if (searchTerm) {
+      return value;
+    }
+
+    return (
+      <HighLight
+        search={searchTerm}
```

Comment:
Since above there is a check 

```
    if (searchTerm) {
      return value;
    }
```
seems like here searchTerm is always expected to be empty? In that case, highlighter is not needed. Please check the logic and search functionality in order to ensure that highlighting feature still works. I found that it does not for file extensions and for job profiles and match profiles by name fields. Please check it carefully comparing to how it works now.

Reply:
Still highlighting does not work for file extensions.
<img width="479" alt="Screen Shot 2019-05-27 at 11 08 41" src="https://user-images.githubusercontent.com/40821852/58405382-f7e1b400-806f-11e9-958d-1dcc208e6cd2.png">